runtime/parsing.c

```diff
@@ -135,13 +135,26 @@ static void print_token(struct parser_tables *tables, int state, value tok)
       fprintf(stderr, "_");
     fprintf(stderr, ")\n");
   }
+  fflush(stderr);
 }
 
 static int trace(void)
 {
   return caml_params->parser_trace || Caml_state->parser_trace;
 }
 
+void print_trace (const char *template, ...){
```

Comment:
This looks much nicer, thank you - it just needs a `static` to stop the symbol-checker tripping.
```suggestion
static void print_trace (const char *template, ...){
```